### PR TITLE
FifoDataFile: Revert PadFile changes from 4.0-1127.

### PR DESCRIPTION
**Pull request by delroth**
Fixes #9006.

Completely untested.

### CODE REVIEW

File: Source/Core/Core/FifoPlayer/FifoDataFile.cpp [hunk 1]
```diff
@@ -55,11 +55,11 @@ bool FifoDataFile::Save(const std::string& filename)
 		return false;
 
 	// Add space for header
-	PadFile(sizeof(FileHeader), file);
+	file.Seek(sizeof(FileHeader), SEEK_END);
 
 	// Add space for frame list
 	u64 frameListOffset = file.Tell();
-	PadFile(m_Frames.size() * sizeof(FileFrameInfo), file);
+	file.Seek(m_Frames.size() * sizeof(FileFrameInfo), SEEK_END);
```
Comment (BhaaLseN) [inline Source/Core/Core/FifoPlayer/FifoDataFile.cpp]: Shouldn't this be ```SEEK_CUR```? Otherwise you're missing ```sizeof(FileHeader)``` in front.

**Comment by delroth** [comment]
This fixes a regression from rc89f04a7 (4.0-1127) which should make this a candidate for 5.0. @Armada651

File: Source/Core/Core/FifoPlayer/FifoDataFile.cpp (hunk 1 above)
Comment (delroth) [inline Source/Core/Core/FifoPlayer/FifoDataFile.cpp]: Fixed.

**Comment by delroth** [comment]
@JMC47 can you test DFF recording with this PR?

**Comment by degasus** [comment]
code LGTM

**Comment by Tilka** [comment]
Oops.

**Pull request merged**